### PR TITLE
Use view binding on talk pages

### PR DESCRIPTION
**Pull request by cooltey**

### CODE REVIEW

File: app/src/main/java/org/wikipedia/talk/TalkTopicActivity.kt [hunk 1]
```diff
@@ -57,55 +58,56 @@ class TalkTopicActivity : BaseActivity(), LinkPreviewDialog.Callback {
 
     public override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_talk_topic)
-        setSupportActionBar(replyToolbar)
+        binding = ActivityTalkTopicBinding.inflate(LayoutInflater.from(this))
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/talk/TalkTopicActivity.kt:61]: Can we just use the Activity's `getLayoutInflater()` function?

File: app/src/main/java/org/wikipedia/talk/TalkTopicsActivity.kt [hunk 2]
```diff
@@ -50,27 +48,28 @@ class TalkTopicsActivity : BaseActivity() {
 
     public override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_talk_topics)
+        binding = ActivityTalkTopicsBinding.inflate(LayoutInflater.from(this))
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/talk/TalkTopicsActivity.kt:51]: same: `getLayoutInflater`

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/talk/TalkTopicActivity.kt (hunk 1 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/talk/TalkTopicActivity.kt]: done,

File: app/src/main/java/org/wikipedia/talk/TalkTopicsActivity.kt (hunk 2 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/talk/TalkTopicsActivity.kt]: Done.

**Pull request merged**